### PR TITLE
Make Crowdin download action generic to avoid nested dependencies issues with allowance flags

### PR DESCRIPTION
**Pull request by hbenali**

### CODE REVIEW

File: .github/workflows/download-crowdin-generic.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Reset translation branch
+        run: |
+          git push origin origin/${{inputs.PROJECT_BRANCH}}:refs/heads/integration/${{inputs.PROJECT_VERSION}}-translation -f
+      - name: Crowdin push
+        uses: crowdin/github-action@v2
+        with:
+          upload_sources: false
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: integration/${{inputs.PROJECT_VERSION}}-translation
```
Comment (bourasom) [inline .github/workflows/download-crowdin-generic.yml:45]: crowdin_branch_name: ${{inputs.CROWDIN_MAINTENANCE_BRANCH_VERSION}}
is a key param. It is not put for develop as it is the main branch and not needed. But it is used for develop-exo, develop-meed and 6.5.x

Comment (hbenali) [reply .github/workflows/download-crowdin-generic.yml:45]: I will take care of it. Thanks

Comment (hbenali) [reply .github/workflows/download-crowdin-generic.yml:51]: For the crowdin branch param I've checked this https://github.com/crowdin/github-action/blob/4eb602e9ff92bc79d92226486ddc39ddc15c83b7/entrypoint.sh#L306 It's OK to put an empty string

Comment (bourasom) [reply .github/workflows/download-crowdin-generic.yml:51]: yes that 's for the main branch not the others

Comment (bourasom) [reply .github/workflows/download-crowdin-generic.yml:51]: We start to use branches in crowdin project

Comment (hbenali) [reply .github/workflows/download-crowdin-generic.yml:51]: PR updated

**Review by bourasom — APPROVED** [review]
LGTM

**Pull request merged**